sherpa/astro/data.py

```diff
@@ -84,107 +84,110 @@ def _notice_resp(chans, arf, rmf):
             arf.notice(bin_mask)
 
 
-def validate_ogip_energy_ranges(rtype, label, elo, ehi, ethresh=None):
-    """Check the lo/hi values are > 0, handling common error case.
+class DataResponse(Data1DInt):
```

Comment:
I'm reluctant to add classes with no documentation, as I've no idea what this is meant to represent. Which will be useful when we look at this in six months time having forgotten this PR.

I assume that the class is an implementation detail - i.e. not really anyone who creates `DataARF` and `DataRMF` classes need to worry about or is going to be exported from this module - but the name is rather generic. Perhaps `DataOGIPResponse`?

Reply:
If it's renamed to include `OGIP` in the class name then the validate method can drop "ogip" - e.g.  `_validate_energy_ranges`